ethsigner/core/src/test/java/tech/pegasys/ethsigner/core/signing/fileBased/FileBasedSignerHelperTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.core.signing.fileBased;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import tech.pegasys.ethsigner.core.signing.TransactionSigner;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.web3j.crypto.CipherException;
+import org.web3j.crypto.WalletUtils;
+
+public class FileBasedSignerHelperTest {
+
+  private static String fileName;
+
+  @BeforeClass
+  public static void createKeyFile() {
+    try {
+      fileName = WalletUtils.generateFullNewWalletFile(MY_PASSWORD, null);
+    } catch (NoSuchAlgorithmException e) {
+    } catch (NoSuchProviderException e) {
+    } catch (InvalidAlgorithmParameterException e) {
+    } catch (CipherException e) {
+    } catch (IOException e) {
+    }
+    new File(fileName).deleteOnExit();
+  }
+
+  private static final String MY_PASSWORD = "myPassword";
+
+  @Test
+  public void testSuccess() throws IOException {
```

Comment:
You'll want to drop the prefix `test` from the method names of all the test methods, as it's not the convention we're using.

Reply:
old habits ...
Done

Reply:
done